homeassistant/components/shelly/sensor.py

```diff
@@ -373,6 +387,30 @@ class RestSensorDescription(RestEntityDescription, SensorEntityDescription):
         entity_category=EntityCategory.DIAGNOSTIC,
         use_polling_wrapper=True,
     ),
+    "humidity_0": RpcSensorDescription(
+        key="humidity:0",
+        sub_key="rh",
+        name="Humidity",
+        native_unit_of_measurement=PERCENTAGE,
+        device_class=SensorDeviceClass.HUMIDITY,
+        state_class=SensorStateClass.MEASUREMENT,
+        entity_registry_enabled_default=True,
+    ),
+    "battery": RpcSensorDescription(
+        key="devicepower:0",
+        sub_key="battery",
+        name="Battery",
+        native_unit_of_measurement=PERCENTAGE,
+        value=lambda status, _: round(status["percent"], 2),
```

Comment:
```suggestion
        value=lambda status, _: status["percent"],
```
This value is already an integer from the device, I used an external power supply to verify multiple values

Reply:
I saw 1 time 3 digit, this is why I rounded. Fine for me to remove.